module-4/backend/week_two.md

```diff
@@ -5,17 +5,29 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is Webpack and why is it useful?
++ It is a tool that compiles your assets together for production. It minimizes the size and number of assets that a client will require
 2. When do you want to use event delegation?
++ When you want to listen for events on child nodes that may not exist at the time that the handler is initialized
 3. What's one difference between ES5 and ES6?
++ constructor syntax
 4. How are you using the MVC design pattern in your Quantified Self project?
++ the database connection is handled through model files. The controller handles routing, and also generating views from the models
 5. How do you execute raw SQL in node?
++ using knex.raw()
 6. What's a callback function and what are some reasons when we use/need callback functions?
++ Callback function is a function that fires off after the completion of another function. Use it by passing a function in as the argument to another function
 7. What is CORS?
++ it is a javascript rule that prevents making calls to servers different than the origin.
```

Comment:
The topic is broader than just JS, and CORS is what *does* allow cross-origin resource sharing (hence the acronym).